docs/guide/snaps-development-guide.md

```diff
@@ -1,14 +1,10 @@
 # Development Guide
 
-::: tip Only Available in MetaMask Flask
-[Snaps](./snaps.html) is only available in [MetaMask Flask](https://metamask.io/flask).
+::: tip Developer Preview Software 
+Snaps is pre-release software. To try Snaps, install [MetaMask Flask](https://metamask.io/flask).
 :::
 
-::: tip Feature Requests
-Do you have feature requests? Other ideas? We'd love to hear about them! [Click here](https://community.metamask.io/c/metamask-flask) to join the discussion.
```

Comment:
Do we not want to keep this? Could just point to our GitHub issues.

Reply:
I do not want to keep this. It doesn't make sense here in the document. Would rather have a call to action like this at the end of the page.